sampler.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	defaultSamplingServerHostPort  = "localhost:5778"
+	defaultSamplingServerHostPort  = "localhost:5778/sampling"
```

Comment:
the `/sampling` is part of the URL, not the server's host-port. I would use a different var if you want it to be configurable